CHANGES

```diff
@@ -1,12 +1,49 @@
-Release 0.2 (released Nov 7, 2017)
+Release 0.3 (released Feb 1, 2018)
 ----------------------------------
 
+.. rubric:: Enhancements
+
+* Added support for intermediate certificates, bundle re-signing and
```

Comment:
actually, basic resigning support came in in 0.2. But really usable it became with 0.3.